src/main/java/org/terasology/launcher/jre/ManagedJRE.java

```diff
@@ -0,0 +1,88 @@
+                        ),
+                        new JREArtefact(17,
+                                new Platform(OS.LINUX, Arch.X64),
+                                "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9.1/OpenJDK17U-jre_x64_linux_hotspot_17.0.9_9.tar.gz",
+                                "c37f729200b572884b8f8e157852c739be728d61d9a1da0f920104876d324733"
+                        ),
+                        // Mac
+                        new JREArtefact(8,
+                                new Platform(OS.MAC, Arch.X64),
+                                "https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u392-b08/OpenJDK8U-jre_x64_mac_hotspot_8u392b08.tar.gz",
+                                "f1f15920ed299e10c789aef6274d88d45eb21b72f9a7b0d246a352107e344e6a"
+                        ),
+                        new JREArtefact(11,
+                                new Platform(OS.MAC, Arch.X64),
+                                "https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.21%2B9/OpenJDK11U-jre_x64_mac_hotspot_11.0.21_9.tar.gz",
+                                "43d29affe994a09de31bf2fb6f8ab6d6792ba4267b9a2feacaa1f6e042481b9b"
+                        ),
+                        new JREArtefact(17,
+                                new Platform(OS.MAC, Arch.X64),
+                                "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9.1/OpenJDK17U-jre_x64_mac_hotspot_17.0.9_9.tar.gz",
+                                "c69b37ea72136df49ce54972408803584b49b2c91b0fbc876d7125e963c7db37"
+                        ),
+                        // Window
+                        new JREArtefact(8,
+                                new Platform(OS.WINDOWS, Arch.X64),
+                                "https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u392-b08/OpenJDK8U-jre_x64_mac_hotspot_8u392b08.tar.gz",
+                                "a6b7e671cc12f9fc16db59419bda8be00da037e14aaf5d5afb78042c145b76ed"
+                        ),
+                        new JREArtefact(11,
+                                new Platform(OS.WINDOWS, Arch.X64),
+                                "https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.21%2B9/OpenJDK11U-jre_x64_windows_hotspot_11.0.21_9.zip",
+                                "a93d8334a85f6cbb228694346aad0353a8cb9ff3c84b5dc3221daf2c54a11e54"
+                        ),
+                        new JREArtefact(17,
+                                new Platform(OS.WINDOWS, Arch.X64),
+                                "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9.1/OpenJDK17U-jre_x64_windows_hotspot_17.0.9_9.zip",
+                                "6c491d6f8c28c6f451f08110a30348696a04b009f8c58592191046e0fab1477b"
+                        )
```

Comment:
I tried for some time to be "smart" about assembling the URL, but that became so much code in the end I don't think it's worth it in a list that we will rarely update or extend.

The main issues are:
- different format of Java version for Java 8 (e.g. `8u392-b08`) and new versions (e.g. `17.0.0+9.1`)
- different ways this version is encoded in the URL
  - for Java 8 it's `jdk8u392-b08` for the jdk path element and `8u392b08` in the artifact version
  - for Java 17 its's URL encoded `17.0.9%2B9.1` in the jdk path element (with full `9.1` build version) and `17.0.9_9`in the artifact version (with only the major build version `9`)
- the major version is also referenced alone

So while we can write some Regex to handle that, compute the correct suffix, etc, I think that just maintaining these lists by hand is probably a better solution.